heureka/ui/src/lib/slices/createAuthDataSlice.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const AUTH_ACTIONS = {
+  SIGN_ON: "signOn",
+  SIGN_OUT: "signOut",
+}
+
+const createAuthDataSlice = (set, get) => ({
+  auth: {
```

Comment:
This goes for all store slices: Please add a ticket for yourself to unit test the slices once you are happy with how the store is designed.

Reply:
Sure, I will